apps/emqx/src/emqx_authentication_config.erl

```diff
@@ -136,7 +136,7 @@ do_pre_config_update({move_authenticator, _ChainName, AuthenticatorID, Position}
 ) ->
     ok | {ok, map()} | {error, term()}.
 post_config_update(_, UpdateReq, NewConfig, OldConfig, AppEnvs) ->
-    do_post_config_update(UpdateReq, check_configs(to_list(NewConfig)), OldConfig, AppEnvs).
```

Comment:
Since `authentication` root field's type is no loner 'lazy',
`NewConfig` is already type checked, so there is no need to check it again here.